modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -130,6 +130,24 @@ For example, if this field is set to 10, you can provision up to 10 times the am
 
 To disable over-provisioning, set this field to 1.
 
+|`thinPoolConfig.chunkSize`
+|`integer`
+|Specifies the statically calculated chunk size for the thin pool. It is used only when the `ChunkSizeCalculationPolicy` field is set to `Static`. The value for this field must be configured in the range 64 KiB to 1 GiB due to the underlying limitations of `lvm2`. 
```

Comment:
```suggestion
|Specifies the statically calculated chunk size for the thin pool. The field is only used when the `ChunkSizeCalculationPolicy` field is set to `Static`. The value for this field must be configured in the range of 64 KiB to 1 GiB because of the underlying limitations of `lvm2`. 
```

Reply:
Updated as per the suggestion.